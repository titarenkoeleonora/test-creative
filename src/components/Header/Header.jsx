const Header = ({toggleMenu}) => {
  return (
    <header className="header">
      <div className="container">
        <div className="header__wrapper">
          <button
            className="header__menu-toggle"
            type="button"
            aria-label="Показать/скрыть меню"
            onClick={toggleMenu}
          >
            Меню
          </button>
        </div>
      </div>
    </header>
  );
};

export default Header;