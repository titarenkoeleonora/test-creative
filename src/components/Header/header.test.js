import React from "react";
import renderer from "react-test-renderer";
import Header from "./Header";

it('Render Header component', () => {
  const tree = renderer
    .create(
      <Header />
    );

  expect(tree).toMatchSnapshot();
});