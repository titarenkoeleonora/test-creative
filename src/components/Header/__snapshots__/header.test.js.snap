// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render Header component 1`] = `
<header
  className="header"
>
  <div
    className="container"
  >
    <div
      className="header__wrapper"
    >
      <button
        aria-label="Показать/скрыть меню"
        className="header__menu-toggle"
        type="button"
      >
        Меню
      </button>
    </div>
  </div>
</header>
`;
